src/pages/PartidoInfo/index.js

```diff
@@ -27,65 +27,52 @@ const PartidoInfo = ({ match }) => {
   }, [id]);
 
   return (
-    <div>
-      <section className="hero is-dark">
-        <div className="hero-body">
-          <div className="container">
-            <h1 className="title is-4">
-              {partido.nome} - {partido.sigla}
+    <div className="divPartido">
+      <div className="infoPartido">
```

Comment:
nesses cards da pagina de partido, usa as colunas do bulma css pra quando for em tablet/celular ele quebrar linha e cada card ficar em 100% um embaixo do outro, qualquer coisa quando eu fizer o PR da minha parte da pra olhar lá como eu fiz, ou só dar uma ligada aqui: https://bulma.io/documentation/columns/responsiveness/